docset/winserver2012-ps/scheduledtasks/Register-ScheduledTask.md

```diff
@@ -16,57 +16,62 @@ Registers a scheduled task definition on a local computer.
 
 ## SYNTAX
 
-### UNNAMED_PARAMETER_SET_1
+### User (Default)
 ```
-Register-ScheduledTask [-TaskName] <String> [[-TaskPath] <String>] [-Action] <CimInstance[]>
- [[-Trigger] <CimInstance[]>] [[-Settings] <CimInstance>] [[-User] <String>] [[-Password] <String>]
- [[-RunLevel] <RunLevelEnum>] [[-Description] <String>] [-AsJob] [-CimSession <CimSession[]>] [-Force]
- [-ThrottleLimit <Int32>]
+Register-ScheduledTask [-Force] [[-Password] <String>] [[-User] <String>] [-TaskName] <String>
+ [[-TaskPath] <String>] [-Action] <CimInstance[]> [[-Description] <String>] [[-Settings] <CimInstance>]
+ [[-Trigger] <CimInstance[]>] [[-RunLevel] <RunLevelEnum>] [-CimSession <CimSession[]>]
+ [-ThrottleLimit <Int32>] [-AsJob] [<CommonParameters>]
 ```
 
-### UNNAMED_PARAMETER_SET_2
+### Xml
 ```
-Register-ScheduledTask [-TaskName] <String> [[-TaskPath] <String>] [-Action] <CimInstance[]>
- [[-Trigger] <CimInstance[]>] [[-Settings] <CimInstance>] [[-Principal] <CimInstance>]
- [[-Description] <String>] [-AsJob] [-CimSession <CimSession[]>] [-Force] [-ThrottleLimit <Int32>]
+Register-ScheduledTask [-Force] [[-Password] <String>] [[-User] <String>] [-TaskName] <String>
+ [[-TaskPath] <String>] [-Xml] <String> [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [<CommonParameters>]
 ```
 
-### UNNAMED_PARAMETER_SET_3
+### Principal
 ```
-Register-ScheduledTask [[-TaskName] <String>] [[-TaskPath] <String>] [-InputObject] <CimInstance>
- [[-User] <String>] [[-Password] <String>] [-AsJob] [-CimSession <CimSession[]>] [-Force]
- [-ThrottleLimit <Int32>]
+Register-ScheduledTask [-Force] [-TaskName] <String> [[-TaskPath] <String>] [[-Principal] <CimInstance>]
+ [-Action] <CimInstance[]> [[-Description] <String>] [[-Settings] <CimInstance>] [[-Trigger] <CimInstance[]>]
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [<CommonParameters>]
 ```
 
-### UNNAMED_PARAMETER_SET_4
+### Object
 ```
-Register-ScheduledTask [-TaskName] <String> [[-TaskPath] <String>] [-Xml] <String> [[-User] <String>]
- [[-Password] <String>] [-AsJob] [-CimSession <CimSession[]>] [-Force] [-ThrottleLimit <Int32>]
+Register-ScheduledTask [-Force] [-InputObject] <CimInstance> [[-Password] <String>] [[-User] <String>]
+ [[-TaskName] <String>] [[-TaskPath] <String>] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob]
+ [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 The **Register-ScheduledTask** cmdlet registers a scheduled task definition on a local computer.
 
-You can register a task to run any of the following application or file types: Win32 applications, Win16 applications, OS/2 applications, MS-DOS applications, batch files (*.bat), command files (*.cmd), or any properly registered file type.
+You can register a task to run executable files (`.exe` and `.com`), batch files (`.bat` and `.cmd`),
+or any registered file type. However, this cmdlet does not check whether the file you intend it to
```

Comment:
can you please review the new lines here? Thank you

Reply:
Oh, this change is the main point of this whole PR. To quote myself:

> Re-wrote a sentence to eliminate the following **arcane** phrases: "Win16 applications", "OS/2 applications", and "MS-DOS applications". These phrases were used [in a sense other than their direct, literal meaning](https://en.wikipedia.org/wiki/Metonymy). Understanding these phrases was in no way essential to understanding this document.

As you can probably understand from the context, these phrases are supposed to refer to "properly registered file type". The problem is that they don't. The reader needs to know a lot of history and arcana before discovering that they are wrong.

For example, I suspect "MS-DOS applications" is a metonym for `.com` files. Of course, MS-DOS apps could have `.exe` extensions too. Had it been Windows 9x or Windows 3.1, the phrase could refer to `.pif` files as well. Sure, you can go learn about NTVDM, NTIO.SYS and NTDOS.SYS, but once did, you discover that all this problematic paragraph intended to say was:

> You can register a task to run any properly registered file type.

Reply:
I was referring to the carrier return that looked strange. Seems that it is fixed now.

Reply:
Huh! Well, if it is fixed now... I'll mark this as resolved.